editor/assets/stylesheets/variables.scss

```diff
@@ -1,6 +1,37 @@
-$gray: #575D65;
-$light-gray: #e0e5e9;
+/* Hugo's new WordPress shades of gray, from http://codepen.io/hugobaeta/pen/grJjVp */
+$black: #000;
+$dark-gray-900: #191e23;
+$dark-gray-800: #23282d;
+$dark-gray-700: #32373c;
+$dark-gray-600: #40464d;
+$dark-gray-500: #555d66;    // use this most of the time for dark items
+$dark-gray-400: #606a73;
+$dark-gray-300: #6c7781;
+$dark-gray-200: #7e8993;
+$dark-gray-100: #8f98a1;
+$light-gray-900: #a2aab2;
+$light-gray-800: #b5bcc2;
+$light-gray-700: #ccd0d4;
+$light-gray-600: #d7dade;
+$light-gray-500: #e2e4e7;   // good for "grayed" items and borders
+$light-gray-400: #e8eaeb;
+$light-gray-300: #edeff0;
+$light-gray-200: #f3f4f5;
+$light-gray-100: #f8f9f9;
+$white: #fff;
 
+/* Extra colors, some from https://make.wordpress.org/design/handbook/foundations/colors/ */
+$blue-wordpress-700: #00669b;
+$blue-wordpress: #0073aa;
+
+$blue-medium: #00a0d2;
+$blue-medium-400: #33B3DB;
+$blue-medium-300: #66C6E4;
+$blue-medium-200: #BFE7F3;
+$blue-medium-100: #E5F5FA;
+
+/* Other */
 $editor-font: "Noto Serif", serif;
+$editor-html-font: "Meno", monospace;
```

Comment:
I expect this font stack to grow further — Menlo is an OSX bundled system font. I will return with a better font stack later, but I don't think we should load any fonts separately.